CacheService/docker_context/AuditHandler.py

```diff
@@ -0,0 +1,82 @@
+from EventTypes import UserCommandEvent, AccountTransaction, ErrorEvent, QuoteServerEvent
+import pickle
+
+
+class AuditHandler:
```

Comment:
this seems to be the old audit handler type that we used in the socket implementation